src/channel/mpmc.rs

```diff
@@ -182,12 +203,29 @@ where
         }
     }
 
+    /// Tries to read the value from the channel without blocking.
```

Comment:
```suggestion
    /// Tries to receive a value from the channel without blocking.
```

Just to be consistent with the function names